From c928aff3d8246b865e50c6ac819c936aad766f05 Mon Sep 17 00:00:00 2001
From: Zheng Yang <zhengyang@rock-chips.com>
Date: Mon, 27 Nov 2017 16:53:46 +0800
Subject: [PATCH] drm: introduce HLG EOTF

CTA-861-G defined Hybrid Log-Gamma (HLG) based on ITU-R
BT.2100-0.

Change-Id: Ieb18284265529ee8d76b250d8bb5b3752425814a
Signed-off-by: Zheng Yang <zhengyang@rock-chips.com>
---
 include/uapi/drm/drm_mode.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/uapi/drm/drm_mode.h b/include/uapi/drm/drm_mode.h
index b4492d50f2ae..33a221fdcf36 100644
--- a/include/uapi/drm/drm_mode.h
+++ b/include/uapi/drm/drm_mode.h
@@ -516,6 +516,7 @@ enum supported_eotf_type {
 	TRADITIONAL_GAMMA_SDR = 0,
 	TRADITIONAL_GAMMA_HDR,
 	SMPTE_ST2084,
+	HLG,
 	FUTURE_EOTF
 };
 
-- 
2.35.3

